packages/e2e-test-app-fabric/test/TextInputComponentTest.test.ts

```diff
@@ -96,39 +96,78 @@ describe('TextInput Tests', () => {
     await component.waitForDisplayed({timeout: 5000});
     const dump = await dumpVisualTree('capitalize-none');
     expect(dump).toMatchSnapshot();
+
     await app.waitUntil(
       async () => {
-        await component.setValue('Hello World');
-        return (await component.getText()) === 'Hello World';
+        await component.setValue('hello world');
+        return (await component.getText()) === 'hello world';
       },
       {
         interval: 1500,
         timeout: 5000,
         timeoutMsg: `Unable to enter correct text.`,
       },
     );
-    expect(await component.getText()).toBe('Hello World');
+    expect(await component.getText()).toBe('hello world');
   });
-  test('TextInputs can autocapitalize: Autocapitalize Sentences', async () => {
+  // Comment out once the sentences mode has been implemented.
+  /*test('TextInputs can autocapitalize: Autocapitalize Sentences', async () => {
     const component = await app.findElementByTestID('capitalize-sentences');
     await component.waitForDisplayed({timeout: 5000});
     const dump = await dumpVisualTree('capitalize-sentences');
     expect(dump).toMatchSnapshot();
-    // Behavior not supported yet.
-  });
-  test('TextInputs can autocapitalize: Autocapitalize Words', async () => {
+
+    // Test behavior when text is set from JS - Requires change in RN Core repo to avoid override issues
```

Comment:
What change is needed in RN core?  Whats the issue here?

Reply:
Previously, I set defaultValues to the TextInputs featuring autocapitalization, which are declared in TextInputSharedExamples.js. Doing this caused an "override error" since that .js file is expected to be an exact copy of its parent living in RN Core. IIRC, the source of conflict was here:

```
\packages\@react-native\tester\overrides.json

"type": "copy",
"directory": "js/examples/TextInput",
"baseDirectory": "packages/rn-tester/js/examples/TextInput",
```